### PR TITLE
update card title

### PR DESCRIPTION
**Pull request by oleksandr-didyk**

### CODE REVIEW

File: WanderLost/WanderLost/Client/Components/Card.razor [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿<CardFrame Title="@Title" Rarity="@TopRarity" ClassName="@ClassName">
+﻿<CardFrame Region="@Region" Zone="@Merchant.Zone" Rarity="@TopRarity" ClassName="@ClassName">
```
Comment (user1736) [inline WanderLost/WanderLost/Client/Components/Card.razor]: property naming should be more detached, for this particular use case `Title` and `Subtitle` should do

File: WanderLost/WanderLost/Client/Components/CardButton.razor [hunk 2]
```diff
@@ -22,10 +22,10 @@
   public Action OnClick { get; set; }
 
   [Parameter]
-  public string ClassName { get; set; } = "button";
+  public string ClassName { get; set; }
 
   [Parameter]
-  public string Title { get; set; } = default!;
+  public string Title { get; set; }
```
Comment (user1736) [inline WanderLost/WanderLost/Client/Components/CardButton.razor]: let's rename this property to `Hint`

File: WanderLost/WanderLost/Client/Components/CardFrame.razor [hunk 3]
```diff
@@ -7,12 +7,21 @@
       <path class="card-frame__border-color" d="m480.95,11.04l-0.1,0c23.9,33.09 54.44,27.7 62.09,24.69c-2.17,16.44 1.83,27.43 27.74,37.58c-17.14,-21.29 -7.17,-32.86 0.15,-37.57c3.85,-2.48 8.39,-3.65 12.96,-3.65c0,0 267.56,-0.43 301.67,-0.43l0.02,145.93c0,47 -23.91,62 -23.91,62l0,1c0,0 23.91,15 23.91,62l0.02,719.53c0,47 -23.9,62 -23.9,62l0,1c0,0 23.9,15 23.9,62l-0.02,187.33l-384.06,0.06l-25.14,-25.23l-29.94,30.3l-29.19,-30.3l-25.08,25.18l-382.61,0.13l-0.02,-187.9c0,-47 23.91,-62 23.91,-62l0,-1c0,0 -23.91,-15 -23.91,-62l0,-719.1c0,-47 23.91,-62 23.91,-62l0,-1c0,0 -23.91,-15 -23.91,-62l0.09,-145.5l299.02,0c4.58,0 9.11,1.17 12.96,3.65c7.33,4.71 17.3,16.28 0.16,37.57c25.91,-10.15 29.91,-21.14 27.74,-37.58c7.64,3.01 38.18,8.4 62.09,-24.69l-0.11,0l-409.9,0.21l-0.28,1335.46l895.62,-0.1l0.02,-1335.51l-415.9,-0.06z" />
     </svg>
     <div class="card-frame__inner">
-        <div class="card-frame__title">@Title</div>
+        <div class="card-frame__title">
```
Comment (user1736) [inline WanderLost/WanderLost/Client/Components/CardFrame.razor]: let's have only 2 divs here

```
<div class="card-frame__title">@Title</div>
<div class="card-frame__subtitle">@SubTitle</div>
```

File: WanderLost/WanderLost/Client/Components/Overlay.razor [hunk 4]
```diff
@@ -11,7 +11,7 @@
 
             @foreach (var Merchant in MerchantGroup.ActiveMerchants.OrderByDescending(x => x.Votes))
             {
-                <Card ClassName="overlay__card" Merchant="@Merchant" SubmitVote="@SubmitVote" CurrentVote="@CurrentVote" />
+                <Card ClassName="overlay__card" Merchant="@Merchant" Server="@Server" Region="@Region"  SubmitVote="@SubmitVote" CurrentVote="@CurrentVote" />
```
Comment (user1736) [inline WanderLost/WanderLost/Client/Components/Overlay.razor]: two spaces :)

**Review by user1736 — CHANGES_REQUESTED** [review]

File: WanderLost/WanderLost/Client/Components/Card.razor (hunk 1 above)
Comment (oleksandr-didyk) [reply WanderLost/WanderLost/Client/Components/Card.razor]: Good point, will change

File: WanderLost/WanderLost/Client/Components/CardButton.razor (hunk 2 above)
Comment (oleksandr-didyk) [reply WanderLost/WanderLost/Client/Components/CardButton.razor]: Sure, makes sense, will do

File: WanderLost/WanderLost/Client/Components/CardFrame.razor (hunk 3 above)
Comment (oleksandr-didyk) [reply WanderLost/WanderLost/Client/Components/CardFrame.razor]: Sure, will do

File: WanderLost/WanderLost/Client/Components/Overlay.razor (hunk 4 above)
Comment (oleksandr-didyk) [reply WanderLost/WanderLost/Client/Components/Overlay.razor]: My bad, will fix

**Pull request merged**